gui/js/csp-fly-to-locations.js

```diff
@@ -9,19 +9,78 @@ class FlyToApi extends IApi {
    * @type {string}
    */
   name = 'flyto';
+// Var for the minimap.
+  wmslayer = null;
+  minimap = null;
+  marker = null;
+// Say the active planet.
+  activePlanet = null;
+// Say the circumfence of the planet.
+  circumfencevar =0;
+
+  // Store last frame's observer position.
+  lastLong = 0.0;
+  lastLat = 0.0;
+  lastHeight = 0.0;
+
+  /**
+   * @inheritDoc
+   */
+  init() {
+    // Show the minimap and make a invisible border.
+    this.minimap = L.map(document.getElementById('minimap'), {
+        center: [52.315625,10.562169],
+        zoom: 3,
+        worldCopyJump:true,
+        maxBoundsViscosity: 0.5,
+        maxBounds:[
+          [-90, -180],
+          [90, 180]
+      ]});
+    this.wmslayer= L.tileLayer.wms('http://maps.sc.bs.dlr.de/mapserv?', {
+      layers: 'earth.bluemarble.rgb'
+    }).addTo(this.minimap);
```

Comment:
I think we could skip the initialization of `this.wmslayer` here, because we don't have the configuration of the mapserver yet. This is passed to this api later, in `configureMinimap`. Then in`configureMinimap` you could do the following:

```js
configureMinimap(mapserver, layer, circumfence) {
  if (this.wmslayer == null) {
    this.wmslayer = L.tileLayer.wms(mapserver, {layers: layer}).addTo(this.minimap);
  } else {
    this.wmslayer.setParams({
      "baseUrl": mapserver,
      "layers": layer
    });
  }
  // Change the circumfence.
  this.circumfencevar = circumfence;
}
```